### ISSUE TITLE
Are we still missing anything relevant from ToBEx

### ISSUE THREAD

**Issue by lynxlynxlynx** [gemrb/gemrb#564]

**Comment by lynxlynxlynx** [comment]
From #426:
>     -ToBEx Level One Proficiency Restrictions (what's this?)
> 
> In BG1 during main character generation fighter class starts with 4 stars of proficiency points but you can only distribute maximum 2 stars per weapon.
IIRC this is handled by a table, but needs checking.

**Comment by burner1024** [comment]
I'm using effect stacking thing in one of my mods... Maybe allocate another bit for this?

**Comment by lynxlynxlynx** [comment]
It would have to be a different bit, so I'm sure it'd be very useful.

**Comment by burner1024** [comment]
Did you mean to say "not sure"?
It's not a problem to set another bit, and there's enough to choose from yet.
Unfortunately, it was introduced for a reason, on classic engine there's nothing that could quite emulate it.
Although I do understand the compatibility implications.

**Comment by lynxlynxlynx** [comment]
Yeah, missing "not", sorry.

**Comment by burner1024** [comment]
I managed to [build](https://github.com/BGforgeNet/TobEx) TobEx.
Given that stacking is not a widely used feature (in fact, I don't know of anyone but me using it..), I probably could relocate the bit.

**Comment by lynxlynxlynx** [comment]
Asked on the forum for more examples:
https://www.gibberlings3.net/forums/topic/31784-mods-using-tobex-effect-stacking/

If none crop up, it will simplify things. But I suspect it was added for SR.

**Comment by burner1024** [comment]
Cool, let's see. I'm not sure it's directly related to poison/regen, though. Those are likely separate fixes.

**Comment by lynxlynxlynx** [comment]
Ok, so no response to that post in five years. Which mod of yours is using stacking and for what?

**Comment by burner1024** [comment]
It's [TnT](https://github.com/BGforgeNet/bg2-tweaks-and-tricks). It uses this feature to unstack Chaos Shield, and for PnP-like protection items.
I've been meaning to expand the usage for a few other components, but hadn't the time.

**Comment by lynxlynxlynx** [comment]
How come you need the extra stacking functionality for that? Can't it be done via https://gibberlings3.github.io/iesdp/opcodes/bg2tob.htm#op206

**Comment by burner1024** [comment]
No, it's a bandaid they used in Fixpack since forever, but that's not how it's supposed to work.
Blocking effectively wastes the second spell/effect. No stacking allows it to keep going when the first one runs out.
